agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -95,10 +96,26 @@ function collect_sos {
 	fi
 }
 
-collect_kernel_config &
-collect_libvirt &
-collect_topology &
-collect_sos &
-collect_block &
+if [[ -n "$sysinfo" ]]; then
+	for item in ${sysinfo//,/ };do
+        	if [[ "$item" == "default" ]] || [[ "$item" == "all" ]]; then
+			collect_kernel_config &
```

Comment:
Given the above comments, we then don't check for all or default here, we just loop over the list and execute them one by one, as the `all` and `default` case testing here is no longer needed.